src/plugins/credential_management/public/service/credential_management_service.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { HttpSetup } from '../../../../core/public';
+import { CredentialCreationConfig, CredentialCreationManager } from './creation';
+
+interface SetupDependencies {
+  httpClient: HttpSetup;
+}
+
+/**
+ * credential management service
+ *
+ * @internal
+ */
+export class CredentialManagementService {
+  credentialCreationManager: CredentialCreationManager;
```

Comment:
I am pretty confused by the structure here, we have `credential management plugin` , `credential management service`, and `credential creation manager`, and seems  `credential management plugin` only has `credential management service`, and `credential management service` only has `credential creation manager`. why do we have so many levels here? If necessary, please add code comments

Reply:
The `credential creation manager` was created to consume configurations related to credential creation. For the initial PR, I removed those and simplified the structure.